### PR TITLE
Added Base Template Files

### PR DESCRIPTION
**Pull request by fru1tstand**
Added base template files, a rudimentary login system, and example usages of both templates (index + dual pane).

Screenshots:
* http://i.imgur.com/ditreOe.png
* http://i.imgur.com/zgPvH0I.png

### CODE REVIEW

File: src/main/scss/_tags.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+// I'll only need padding and margin when I say I need it.
+html, body,
+h1, h2, h3, h4, h5, h6, p
+ol, ul, li, dd, dl,
+table, thead, tbody, tfoot, th, tr, td {
+	padding: 0;
+	margin: 0;
+}
+
+html {
```
Comment (jonathanpli) [inline src/main/scss/_tags.scss]: could set both html and body to height: 100%

File: src/main/java/com/jonathanpli/exceptionnull/controller/IndexController.java [hunk 2]
```diff
@@ -3,26 +3,54 @@
...
+	// TODO(#15): Remove when proper login/logout is implemented.
+	/**
+	 * A simple login
+	 */
+	@RequestMapping(value = "/login")
+	public ModelAndView login(
+			@RequestParam(value = "login") String login,
+			HttpServletRequest request) {
+		if (login.equals(PASSWORD)) {
+			request.getSession().setAttribute(SESSION_IS_LOGGED_IN, "");
```
Comment (jonathanpli) [inline src/main/java/com/jonathanpli/exceptionnull/controller/IndexController.java]: empty string counts as having the attribute set? Interesting. Could we set it to true for clarity?

File: src/main/resources/templates/index.html [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
+{% block body %}
+	<div class="container">
+		Hi, there is content in here.
+	</div>
+	<div class="container">
+		<ul>
+			<li>Visit <a href="/login?login=...">/login?login=...</a> to log in (you must
+				provide the correct password</li>
+			<li>Visit <a href="/logout">/logout</a> to log out</li>
+			<li>Visit <a href="/learn/a language/a module/edit">/learn/a language/a module/edit</a>
```
Comment (jonathanpli) [inline src/main/resources/templates/index.html:15]: For the demo, each one could have a separate tag (temporarily)

**Review by jonathanpli — CHANGES_REQUESTED** [review]

File: src/main/java/com/jonathanpli/exceptionnull/controller/IndexController.java (hunk 2 above)
Comment (fru1tstand) [reply src/main/java/com/jonathanpli/exceptionnull/controller/IndexController.java]: Done.

File: src/main/resources/templates/index.html (hunk 3 above)
Comment (fru1tstand) [reply src/main/resources/templates/index.html:15]: Talked offline, the href is a single url.

File: src/main/scss/_tags.scss (hunk 1 above)
Comment (fru1tstand) [reply src/main/scss/_tags.scss]: Done.

**Comment by fru1tstand** [comment]
Done.

**Review by jonathanpli — APPROVED** [review]

**Pull request merged**